lib/bolt/outputter/human.rb

```diff
@@ -112,9 +113,11 @@ def print_result(result)
               @stream.puts(indent(2, "STDERR:"))
               @stream.puts(indent(4, result['stderr']))
             end
-          else
+          elsif result.generic_value.any?
             @stream.puts(indent(2, ::JSON.pretty_generate(result.generic_value)))
           end
+        else
+          @stream.puts(indent(2, "#{result.action.capitalize} completed successfully with no result"))
```

Comment:
Since `#print_result` is used by more than just tasks, I felt it was best to use the action from the result instead of printing "Task completed successfully..."